.github/workflows/php.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - "7.3"
+          - "7.4"
+        dependencies:
+          - "lowest"
+          - "highest"
+        include:
+          - php-version: "8.0"
+            dependencies: "locked"
+            composer-options: "--ignore-platform-reqs"
```

Comment:
since magento/composer already supports PHP 8 (at least it's declared), do we still need this parameter?

Reply:
Updated